main.c

```diff
@@ -1,8 +1,8 @@
-/* Standard Headers */
```

Comment:
Leave them capital only, plz!
And update your branch before making changes in "main.c".
This is an older version of "main,c", it was changed is recent commits.